### PR TITLE
[fingerpose] Add types and tests for poseData

### PR DESCRIPTION
**Pull request by EthanL06**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/andypotato/fingerpose/blob/cbafab4f5811ddd22d52a66d28cdfb4c59df9358/src/GestureEstimator.js#L30C1-L34C10>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/fingerpose/index.d.ts [hunk 1]
```diff
@@ -109,7 +109,7 @@ export class GestureEstimator {
         keypoints3D: Keypoint3D[],
         minScore: number,
     ): {
-        poseData: GestureDescription[];
+        poseData: Array<[string, string, string]>;
```
Comment (peterblazejewicz) [inline types/fingerpose/index.d.ts:112]: please consider:
```suggestion
        poseData: Array<[name: string, curl: string, direction: string]>;
```
named tuples are widely supported now.
image

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@EthanL06 thanks!

**Comment by EthanL06** [comment]
Ready to merge

**Comment by EthanL06** [comment]
Ready to merge

**Pull request merged**